pkg/basicserver/metrics.go

```diff
@@ -17,15 +17,24 @@ package server
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
-	// ServerMaxProcsGauge record the maxprocs.
+	// ServerMaxProcsGauge records the maxprocs.
 	ServerMaxProcsGauge = prometheus.NewGauge(
 		prometheus.GaugeOpts{
 			Namespace: "pd",
-			Subsystem: "service",
+			Subsystem: "server",
```

Comment:
Oh, I noticed that it will change the current behavior.

Reply:
> Oh, I noticed that it will change the current behavior.

The naming change is to be consistent with tidb server.

Reply:
The tso/scheduling service will share these metrics, if we call it `pd_server_*`, it might be strange. And I think there is no need to keep consistent with the tidb server since the name is already different from `tidb_server_*`.

Reply:
> The tso/scheduling service will share these metrics, if we call it `pd_server_*`, it might be strange. And I think there is no need to keep consistent with the tidb server since the name is already different from `tidb_server_*`.

Well, need me to change it back to "server"?

Reply:
Yes, I prefer to change it back to pd_service_*

Reply:
> Yes, I prefer to change it back to pd_service_*

Done, thx.